content/en/getting_started/integrations/google_cloud.md

```diff
@@ -0,0 +1,390 @@
+
+[Traces][31] collected through the Trace Agent enable comprehensive Application Performance Monitoring (APM), helping you understand end-to-end service performance. 
+
+For the full list of benefits of installing the Agent on your cloud instances, see [Why should I install the Datadog Agent on my cloud instances?][27]
+
+
+
+## Explore related services
+
+### Private Service Connect 
+**(Only available in US5 & US1)**
+
+[Private Service Connect (PSC)][45] is a Google Cloud networking product that enables you to access [Google Cloud services][43], [third-party partner services][44], and company-owned applications directly from your Virtual Private Cloud (VPC).
+
+Get started with the [Google Cloud Private Service Connect guide.][46]
+
+### Google Cloud Run
+
+To begin monitoring serverless functions running on Google Cloud Platform, enable the [Google Cloud Platform][6] integration. 
+
+The [Google Cloud Run][34] page provides a list of out-of-the-box metrics collected by the Google Cloud integration, along with instructions for [Log Collection through Dataflow][35].
+
+### Cloud Cost Management (CCM)
+
+[Cloud Cost Management][50] provides insights for engineering and finance teams to understand how infrastructure changes impact costs, allocate spend across your organization, and identify potential improvements.
+
+To use Google Cloud Cost Management in Datadog, [follow these steps][51].
+
+### Security
+
+### Cloud SIEM
+
+Cloud SIEM provides real-time analysis of operational and security logs, while using out-of-the-box integrations and rules to detect and investigate threats.
+To use this feature, see [Getting Started with Cloud SIEM][36].
+
+To view security findings from [Google Cloud Security Command Center][37] in Cloud SIEM, toggle the **Enable collection of security findings** option under the **Security Findings** tab & follow the setup instructions on the [Google Cloud Security Command Center guide][38].
+
+{{< img src="integrations/google_cloud_platform/security_findings.png" style="width:90%;" >}}
+
+### Cloud Security Management
+
+Datadog Cloud Security Management (CSM) delivers real-time threat detection and continuous configuration audits across your entire cloud infrastructure.
+Check out the [Setting up Cloud Security Management guide][39] to get started.
+
+After setting up CSM, toggle the **Enable Resource Collection** option under the **Resource Collection** tab to start collecting configuration data for the [Resource Catalog][41] and CSM. Then, follow these instructions to enable [Misconfigurations and Identity Risks (CIEM)][40] on Google Cloud.
+
+{{< img src="integrations/google_cloud_platform/gcp_resource_collection.png" style="width:100%;" >}}
+
+### Database Monitoring (DBM)
+Use [Database Monitoring (DBM)][49] to gain increased insight on performance metrics, host health and query samples for your Google Cloud SQL databases.
```

Comment:
do we need this?

Reply:
I guess not, I just thought it might be relevant for some customers since DBM is growing.